examples/standalone-in-docker/main.tf

```diff
@@ -70,6 +70,26 @@ resource "mongodb_opsmanager" "opsman" {
       "mms.mail.ssl"                  = "false"
       "automation.versions.directory" = "/data/automation/mongodb-releases"
       "automation.versions.source"    = "mongodb"
+      "automation.agent.version"      = "10.2.0.5851-1"
+    }
+  }
+}
+
+resource "mongodb_automation_agent" "automation_agent" {
+  host {
+    user     = "root"
+    hostname = "127.0.0.1"
+    port     = docker_container.mdb0-0.ports[0].external
+  }
+
+  automation {
+    binary     = "${mongodb_opsmanager.opsman.opsmanager[0].central_url}/download/agent/automation/mongodb-mms-automation-agent-10.2.0.5851-1.linux_x86_64.tar.gz"
+    baseurl    = mongodb_opsmanager.opsman.opsmanager[0].central_url
```

Comment:
was following the same pattern as opsmanager_config.go -> short version as name, long version in tag : "automation:mmsBaseUrl" -- can't reuse it directly anyway will still need to have the tag due to camel-case. Could change that so they all match like this, but once we get to mmsGroupId I disagree since publicly we say project and not group, including in the API now.

Reply:
Project and Group are two interchangeable terms (as per our docs); however, the parameter's name (in AA's local.config file) is called `mmsGroupId`.

Imagine a situation where a user is trying to debug a configuration that the terraform-provider has generated.  I look at this as, what would be the most natural thing for me to get if I knew nothing about this project and I would be trying to use it?

1. `mmsGroupId (local.config) == mms_group_id (terraform)`
2. or `mmsGroupId (local.config) vs. project_id (terraform)`

A few questions I would ask myself if presented with something like this, are:
  - What is a project_id? would have to find it (and read it) in the public-facing docs -> slow
  - where is project_id used?
  - where did I get this hash-looking value for `mmsGroupId` from?
  - then I'd probably figure out it's the same thing as mmsGroupId
  - or if I wanted to double-check, I'd have to grok the provider's codebase

The more terms which represent the 'same thing', deviate from one another, the more intrinsic cognitive load is put on the user.  

It's easy to infer camel-case vs. snake-case (probably because of [our brains](https://www.mnn.com/lifestyle/arts-culture/stories/why-your-brain-can-read-jumbled-letters); different nouns... is harder!

Reply:
changed